canonical/canonical.com
# Remove stale templates and styles

## Done

- Removed stale templates and corresponding styles
- Drive-by's:
  - Updated partners site map
  - Fixed indentation 
  - Added redirects

## Notes (please read before reviewing)

There are four stale surus that are still in use and as such were not removed in this PR:

- `p-strip--suru`, used in https://canonical-com-978.demos.haus/blog/archives 
- `p-strip--suru-topped` used in /careers/thank-you:
![Screenshot from 2023-07-19 18-31-35](https://github.com/canonical/canonical.com/assets/17607612/c0f9f17c-b836-430b-9774-198fd8fbf605)
- `p-strip--sutu-top` used in https://canonical-com-978.demos.haus/partners/request-login
- `p-strip--suru-background` used in https://canonical-com-978.demos.haus/partners/isv-partners

I believe these pages are still needed so will update them according to updated designs in a separate PR.

There are also two stale base templates that are still needed for now:
-  `/careers/base-template` 
- `/careers/base-tabs`

To give some context,  these used to be the base templates for the old department pages/all roles page which included almost all of careers content as tabbed content. The department pages should not extend either if these anymore, but in order to update this the back end views which render these templates need to be refactored. I'll do this in a separate PR as well. 

## QA 

- Check out this feature branch
- Run the site using the command `./run serve`
- Visit the following list of removed templates and see that you are properly redirected: 
  - https://canonical-com-978.demos.haus/careers/ethics 
  - https://canonical-com-978.demos.haus/careers/lifestyle
  - https://canonical-com-978.demos.haus/partners/apps-and-snaps
  - https://canonical-com-978.demos.haus/partners/devices-and-iot
  - https://canonical-com-978.demos.haus/partners/isv-partners
  - https://canonical-com-978.demos.haus/partners/kubernetes
- Check that removing the following partials does not affect existing pages. There should be no visual or functional changes when compared against the live site:
  - `/careers/_vacancies.modal.html` check against careers pages and career explorer on live site. 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5153
